packages/zambdas/package.json

```
@@ -3,10 +3,9 @@
   "version": "1.4.11",
   "private": true,
   "scripts": {
-    "start": "ENV=${ENV:-local} export HOME=./ && npm run setup-deployed-resources ${ENV:-local} && env-cmd -f ./.env/${ENV:-local}.json sls offline --stage ${ENV:-local}",
+    "start": "ENV=${ENV:-local} export HOME=./ && npm run setup-deployed-resources ${ENV:-local} && nodemon --watch 'src/**/*.ts' --exec 'tsx' src/local-server/index.ts -- env=.env/${ENV:-local}.json",
```

no more need in HOME=./ cause there's no more serverless
```suggestion
    "start": "ENV=${ENV:-local} npm run setup-deployed-resources ${ENV:-local} && nodemon --watch 'src/**/*.ts' --exec 'tsx' src/local-server/index.ts -- env=.env/${ENV:-local}.json",
```